src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs

```diff
@@ -17,13 +17,12 @@ public class AssemblyLoadContextTest
         private const string TestAssembly = "System.Runtime.Loader.Test.Assembly";
 
         [Fact]
-        [ActiveIssue(15101)]
         public static void GetAssemblyNameTest_ValidAssembly()
         {
             string originalDir = Environment.CurrentDirectory;
             try
             {
-                Environment.CurrentDirectory = AppContext.BaseDirectory;
+                Environment.CurrentDirectory = Path.GetDirectoryName(typeof(string).Assembly.Location);
```

Comment:
Setting CurrentDirectory affects things process-wide.  Is it possible this is going to interfere with other tests in the same assembly?  By default, tests in different test classes may run concurrently.